braidpool_spec.md

```diff
@@ -260,13 +406,39 @@ reject it for the following reasons:
    in the absence of attackers, cohorts exceeding the k-width happen naturally
    and cannot be prevented.
 
+[kp]
+
+If we want to do a literature review, we should include other proposals
+too.
+
+[/kp]
+
 ## Simple Sum of Descendant Work
 
+[kp]
+
+I still don't see why we still need it. We needed this when in your original
+proposal the beads were not candidate bitcoin blocks. If all beads on braidpool
+are candidate bitcoin blocks, then we don't need the SSDW algorithm.
+
+[/kp]
+
```

Comment:
For resolving double-spends among share-chain transactions (not bitcoin transactions).